src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/ModuleSmoothCamera.kt

```diff
@@ -31,37 +30,52 @@ import net.minecraft.util.math.Vec3d
  * Makes your camera move smoother.
  */
 object ModuleSmoothCamera : ClientModule("SmoothCamera", Category.RENDER) {
+    private val enableFirstPOV by boolean("Enable First Pov", default = false)
+    private val resetOnPerspectiveChange by boolean("Reset on Perspective Change", default = true)
```

Comment:
```suggestion
    private val enableFirstPOV by boolean("EnableFirstPOV", default = false)
    private val resetOnPerspectiveChange by boolean("ResetOnPerspectiveChange", default = true)
```

the theme automatically adds the spaces when you have the setting enabled